.gitattributes

```diff
@@ -1,3 +1,4 @@
 gradlew linguist-generated=true
 gradlew.bat linguist-generated=true
 config/bin/* filter=lfs diff=lfs merge=lfs -text
+**/snapshots/**/*.png filter=lfs diff=lfs merge=lfs -text
```

Comment:
I added this line to .gitattributes similar to the circuit [PR](https://github.com/slackhq/circuit/pull/204) I referenced. I also modified the folder name to be snapshots to match.